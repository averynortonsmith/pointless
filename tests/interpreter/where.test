
output = [x, y, z, a, b] |> println

x = x where x = 3 
y = (a + b where a = 3) where b = 2 
z = id where id(n) = n
a = x + y where {
  (x, y) = ("asdf", "")
}
b = a where foo = "bar"