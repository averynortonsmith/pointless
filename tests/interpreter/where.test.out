[3, 5, PtlsFunc, "asdf", "asdf"]
